src/services/authServices.js

```diff
@@ -94,27 +95,32 @@ class authService {
      * @returns
      * @author khushbuw
      */
-    static async generatePassword(id, requestParams,user) {
+    static async generatePassword(id, requestParams, userRole) {
         try {
-            if(requestParams.password == '' || requestParams.cpassword == ''){
-                return {error:config.emptyFields}
+            if ((userRole == 'customer') || (userRole == 'Customer')) {
```

Comment:
Can you please explain why you have checked with a hard-coded string values ?

Reply:
Please use toLowercase function to avoid OR condition. Also check your entire code for this change.